custom/icds_reports/management/commands/sql_scripts/nos_of_deaths.sql

```diff
@@ -0,0 +1,17 @@
+
+SELECT
+  convert_to(state, 'UTF8') AS state,
```

Comment:
This should already be utf-8 coming out of the db. More specifically the python should already be able to handle whatever comes out